protocol/rpcconsumer/rpcconsumer.go

```diff
@@ -247,10 +248,25 @@ rpcconsumer 127.0.0.1:3333 COS3 tendermintrpc 127.0.0.1:3334 COS3 rest <flags>`,
 			}
 			// handle flags, pass necessary fields
 			ctx := context.Background()
-			networkChainId, err := cmd.Flags().GetString(flags.FlagChainID)
+			var networkChainId string
+			cmdChainIdFlag := viper.GetString(flags.FlagChainID)
+			clientTomlConfig, err := config.ReadFromClientConfig(clientCtx)
 			if err != nil {
 				return err
 			}
+			if cmdChainIdFlag != app.Name {
+				networkChainId = cmdChainIdFlag
+			} else {
+				if clientTomlConfig.ChainID != "" {
+					networkChainId = clientTomlConfig.ChainID
+				} else {
+					networkChainId = app.Name
+				}
+			}
+			if networkChainId != app.Name {
+				utils.LavaFormatInfo("ChainID is different from " + app.Name)
```

Comment:
lets remove it as our app.name is lava and testnet is lava-testnet-1 so it will push this always 
instead we can print the chain id that was set 

utils.LavaFormatInfo("Running with chain-id:" + chainId)

Reply:
fixed: [106ac4b](https://github.com/lavanet/lava/pull/478/commits/106ac4b912d522d204c87c21d03b94417c521858)